components/root/root.jsx

```diff
@@ -542,6 +543,7 @@ export default class Root extends React.PureComponent {
                         )}
                         <ModalController/>
                         <GlobalHeader/>
+                        <CloudEffects/>
```

Comment:
These need to run regardless of the product the user is in, so placed them here. Made CloudEffects itself as lightweight as possible (only checks whether user is logged in and the instance is cloud) to not impact performance for non-cloud and non-admin users.